python/lsst/ip/isr/brighterFatterKernel.py

```diff
@@ -343,9 +343,32 @@ def fromTable(cls, tableList):
 
         amps = ampTable['AMPLIFIER']
 
-        expIdMaskList = ampTable['EXP_ID_MASK']
-        rawMeanList = ampTable['RAW_MEANS']
-        rawVarianceList = ampTable['RAW_VARIANCES']
+        # Determine version for expected values.  The ``fromDict``
+        # method can unpack either, but the appropriate fields need to
+        # be supplied.
+        calibVersion = metadata.get('bfk_VERSION', 1.0)
```

Comment:
line 234 does not do this logic. It seems to assume that the version is going to always be there.

Reply:
Switched to be a direct get.  If this doesn't exist, than there's a larger issue.

Reply:
The logic duplication is something I didn't see an obvious way to avoid.  I've tried to keep the `fromDict` logic as the place that does transformations (`mean` -> `rawMean`, etc), as that is used by both the FITS and yaml readers.  This ensures that `fromTable` only needs to do simple repacking of fields for the two versions.